opensrp-chw-core/src/main/java/org/smartregister/chw/core/activity/CoreChildProfileActivity.java

```diff
@@ -473,6 +473,16 @@ public void setVisitAboveTwentyFourView() {
         textViewRecord.setTextColor(getResources().getColor(R.color.light_grey_text));
     }
 
+    @Override
+    public void setVisitDone() {
+        textViewNotVisitMonth.setText(getString(R.string.visit_done));
```

Comment:
Can you confirm this text view is the right one? **textViewNotVisitMonth**, the naming and the text set are a bit confusing